### PR TITLE
Alerting: Add placeholder to the Email Contact Point Message

### PR DESCRIPTION
**Pull request by ppcano**
Discussed and agreed by @konrad147 

This PR adds a placeholder to the message input of the Email Contact Point, indicating the default value that will be used. 

**Before**

Screenshot 2024-03-07 at 13 07 30

**Now**

Screenshot 2024-03-07 at 13 02 54

This minor change clarifies the connection between contact point templating properties and notification templates. 

Other contact points already display the placeholder value-this one seems to be missing.

### CODE REVIEW

**Review by yuri-tceretian — APPROVED** [review]

**Comment by ppcano** [comment]
@yuri-tceretian, following up on this one, should I merge it?

**Review by grobinson-grafana — APPROVED** [review]

**Comment by yuri-tceretian** [comment]
@ppcano yes, once PR is approved you can merge it unless reviewer did not ask for a second eyes or you think some specific person should take a look.

**Pull request merged**
